### PR TITLE
[NP-4997] Add labels property to Capability model

### PR DESCRIPTION
**Pull request by Mykola97**
https://nanopay.atlassian.net/browse/NP-4997

### CODE REVIEW

File: src/foam/nanos/fs/FileCapabilityLabelDAODecorator.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+foam.CLASS({
+  package: 'foam.nanos.fs',
+  name: 'FileCapabilityLabelDAODecorator',
+  extends: 'foam.dao.ProxyDAO',
+  documentation: 'Decorator to add labels from capability to file',
+  javaImports: [
+    'foam.dao.DAO',
+    'foam.nanos.crunch.UserCapabilityJunction',
+    'foam.nanos.fs.File',
+    'net.nanopay.crunch.document.Document'
```
Comment (lchanmann) [inline src/foam/nanos/fs/FileCapabilityLabelDAODecorator.js]: crunch document is nanopay's model.

File: src/foam/nanos/crunch/document/Document.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.document',
+  name: 'Document',
```
Comment (lchanmann) [inline src/foam/nanos/crunch/document/Document.js:10]: Are we moving crunch Document from nanopay to FOAM? if so, will need to update references on nanopay side and would also need migration.

Comment (KolombetM) [reply src/foam/nanos/crunch/document/Document.js:10]: It was suggested by Joel. We didn't find another way to make code less brittle.

Comment (jlhughes) [reply src/foam/nanos/crunch/document/Document.js:10]: I'm not sure how we can migrate the data.  I believe we'll have to keep both so we can load and copy.

**Comment by lchanmann** [comment]
@Mykola97 if you have the nanopay PR for moving crunch Document to foam please add a link in the description.

**Review by lchanmann — APPROVED** [review]

**Comment by lchanmann** [comment]
@jlhughes @kgrgreer please review.

**Review by jlhughes — COMMENTED** [review]
LGTM

**Comment by kgrgreer** [comment]
Nice job.

**Pull request merged**